### PR TITLE
Referencing_documentation_other_libraries

### PR DESCRIPTION
**Pull request by jposada202020**
As title mentions

### CODE REVIEW

**Comment by jposada202020** [comment]
@kattni I removed the title, I guess after all these years people stop using this, so now we only have the module name. I have added information, that I would like if possible gets validated, in how to show this information.

**Review by kattni — APPROVED** [review]

File: docs/design_guide.rst [hunk 1]
```diff
@@ -383,6 +406,14 @@ Renders as:
 
   :param float degrees: Degrees to turn right
 
+Documentation References to other Libraries
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+When you need to make references to documentation in other libraries you should refer the class using single
+backticks  ``:class:`~adafruit_motor.servo.Servo```. You must add also the reference in the ``conf.py`` file in the
```
Comment (kattni) [inline docs/design_guide.rst:412]: Thank you for adding this! 

@jposada202020 You could update this to read "`You must also add`". It flows a little better.

**Comment by jposada202020** [comment]
Thanks, maybe we should discuss this resource to people, at least that people check this guide before create any library to see if any changes have been made, and also a refresh for the Librarians :)

File: docs/design_guide.rst (hunk 1 above)
Comment (jposada202020) [reply docs/design_guide.rst:412]: No problem, will do,

File: docs/design_guide.rst [hunk 2]
```diff
@@ -383,6 +406,14 @@ Renders as:
 
   :param float degrees: Degrees to turn right
 
+Documentation References to other Libraries
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+When you need to make references to documentation in other libraries you should refer the class using single
+backticks  ``:class:`~adafruit_motor.servo.Servo```. You must also add the reference in the ``conf.py`` file in the
+``intersphinx_mapping section`` adding a new entry::
```
Comment (kattni) [inline docs/design_guide.rst:413]: This should read `by adding a new entry`.

Comment (jposada202020) [reply docs/design_guide.rst]: Changed in https://github.com/adafruit/circuitpython/pull/4652/commits/cb42acea53d4780a018db3bea253708d5d7dd404

File: docs/design_guide.rst (hunk 1 above)
Comment (jposada202020) [reply docs/design_guide.rst]: Changed in https://github.com/adafruit/circuitpython/pull/4652/commits/a512dd230df419c204b7acd49c0cc7f9ebdce290

**Review by kattni — APPROVED** [review]

**Pull request merged**